homeassistant/components/tuya/switch.py

```diff
@@ -433,6 +433,21 @@
             entity_category=ENTITY_CATEGORY_CONFIG,
         ),
     ),
+    # Fingerbot
+    # 
+    "szjqr": (
+        SwitchEntityDescription(
+            key=DPCode.SWITCH,
+            name="Finger Up",
```

Comment:
Should we just call it, Switch?
As there is no official documentation of this device category, it means we also don't know if this re-used for other devices as well.

Reply:
There are two move modes - click and switch. In click mode, the "finger" goes down then stops for delay time and goes up. The value of this property returns to "true" after a while. In switch mode, the value of the property corresponds to the position of the "finger". If we name it as simply "switch" it will be not clear what it switches. Here is a screenshot of device page in mobile app. 
![image](https://user-images.githubusercontent.com/26710811/142430023-7ad7beed-c294-4408-bcb1-5783ab0d99fe.png)

Reply:
> Here is a screenshot of device page in mobile app.

The app is customizable for every single device (and even per distributor) and does not reflect the API at full.

The comment stays the same, since we have no specific device information, we might be better off choosing a more generic name in this case. Can we come up with something?

Reply:
Maybe "Position"? It's a bit more describes physical aspects of the device.

Reply:
You can't really turn on/off a position :) (as this is represented as a switch in Home Assistant).

Reply:
Ok, let it be "Switch"